### PR TITLE
2.18 release notes and blog post

### PR DESCRIPTION
**Pull request by dpobel**

### CODE REVIEW

File: changelog/front-commerce-2.18.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- **RMA:** properly created date with the corresponding timezone
+- **Routes:** don't loose dynamic routes with same name than subdirectory
+- **Routes:** prioritize more specific children routes of a dynamic path
+- **Social login:** merge guest and user cart on login with Magento2
+
+### Features
+
+- **BigCommerce:** add an image proxy to serve resized and optimized images
+- **BigCommerce:** reset my password
+- **GraphQL:** remove FRONT_COMMERCE_WEB_UNSAFE_USE_DEPRECATED_FIELDS env. Gsupport
```
Comment (real34) [inline changelog/front-commerce-2.18.mdx:85]: what did you mean here?

File: changelog/front-commerce-2.18.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+- **UX:** prevent event propagation on clicks on a pending button
+- **Front-Commerce Prismic module:** add default preset format for image transformer
+
+<BackportList
```
Comment (real34) [inline changelog/front-commerce-2.18.mdx:89]: :heart_eyes:

**Review by real34 — APPROVED** [review]

File: changelog/front-commerce-2.18.mdx (hunk 1 above)
Comment (dpobel) [reply changelog/front-commerce-2.18.mdx:85]: me nothing :) I'm only the copy pasting :robot: 

joke aside, there's an extra `G` :)

Comment (real34) [reply changelog/front-commerce-2.18.mdx:85]: Ah yes, that's simple in fact…

Comment (real34) [reply changelog/front-commerce-2.18.mdx:85]:
```suggestion
- **GraphQL:** remove FRONT_COMMERCE_WEB_UNSAFE_USE_DEPRECATED_FIELDS env. support
```

**Pull request merged**